src/styles/globalStyle.ts

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/react';
+import normalize from 'emotion-normalize';
+import { colors } from './theme';
+
+const globalStyle = css`
+  ${normalize}
+
+  @font-face {
+    font-family: 'Pretendard';
+    font-weight: 700;
+    src: url('styles/fonts/Pretendard-Bold.woff');
```

Comment:
local 을 추가하는 건 어떨까요? 사용자의 컴퓨터에 이미 있는 폰트라면 다운 받지 않고 사용자의 컴퓨터에 있는 폰트를 사용하게 하는 속성입니다.

예시입니다.

```
src: local('Pretendard Thin'), url('/fonts/Pretendard-Thin.woff2') format('woff2');
```

Reply:
오 이런게 있군요! 몰랐는데 알려주셔서 감사합니다 ㅎㅎ
검색해보니 `IE 8 이하 브라우저에서는 local() 속성을 인식하지 못하고, 사용자 로컬에 폰트가 잘못 설치된 경우 원하는 화면이 노출되지
않을 수도 있다`는데 혹시 사용하실때 관련 이슈를 겪으신 적이 있으신가요?? 궁금해서 여쭈어봅니다!
추가로, 저희는 아이폰의 웹 뷰 작업을 진행중인데 모바일 기기에서도 동일하게 적용되는지도 궁금하네요 ㅎㅎ

Reply:
아 우선 제가 IE 환경 대응을 안 한지가 오래되어서 해당 이슈는 직접 겪어보진 못했습니다.

예전에 IE환경이면 chrome으로 띄우는 작업을 했어가지고 IE는... 네 ㅠ 경험해보지 못한 이슈입니다!

> 저희는 아이폰의 웹 뷰 작업을 진행중인데 모바일 기기에서도 동일하게 적용되는지도 궁금하네요 ㅎㅎ

경험해보지 못한 이슈인데, 웹뷰에서도 동일하게 local 폰트를 찾는지는 잘 모르겠습니다 ㅠ 찾아봐도 레퍼런스를 찾아보기 어렵네요...

제 추측상은 안 될 것 같긴합니다. 영지님두 RN 개발해보셔서 아시겠지만 앱에서 폰트 설정을 하는 네이티브 코드를 일일이 작성해줬었던 경험을 되짚어보면,, 유연하게 사용자 핸드폰에 설치된 폰트를 읽어오는 작업은 안 해줄 것 같아요

Reply:
그쵸 ㅠㅠ 저도 IE 대응은 안하는데 정말 단순한 궁금증에 여쭈어보았네요!
웹 뷰에서 local 폰트 찾는지 레퍼런스를 저도 못찾아서 GPT한테 물어봤는데 아래와 같이 대답해주네요!
실제로 동작하는지 테스트할 방법이 있을지 모르겠네요. ㅠㅠ
일단 적용해보고 이슈 생기면 공유 드리겠습니다 ㅎㅎ 좋은 정보 감사합니다!

## GPT 답변
Yes, the approach of using the `local()` value in the `@font-face` rule works on iPhones as well. When you set the `src` property to `local('FontFileName')`, the web view on an iPhone will attempt to find and use the specified font if it is installed on the device.

Here are the general steps to follow:

1. Make sure the font file is installed on the iPhone device. This can be done by installing a custom font through an iOS configuration profile or by including the font file within a specific iOS app.

2. Use the `@font-face` rule in the CSS of your web view to define the font and specify the `src` property as `local('FontFileName')`, where `'FontFileName'` is the name of the installed font.

3. Apply the font to the desired elements using CSS by setting the `font-family` property to the name specified in the `@font-face` rule.

When the web view is loaded on an iPhone with the specified font installed, it will retrieve and use the font file from the device. If the font is not available, it will fall back to a default font according to the defined `font-family` hierarchy.

Keep in mind that the availability of fonts and their installation methods may depend on the iOS version and restrictions set by Apple.